segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/StreamSegmentContainer.java

```diff
@@ -174,6 +179,9 @@
                 this.executor, this.traceObjectId);
         shutdownWhenStopped(this.metadataCleaner, "MetadataCleaner");
         this.metrics = new SegmentStoreMetrics.Container(streamSegmentContainerId);
+        this.containerEventProcessor = new ContainerEventProcessorImpl(streamSegmentContainerId,
+                getOrCreateContainerProcessorSegment(streamSegmentContainerId, Duration.ofSeconds(10)), this.executor);
```

Comment:
do not hardcode timeouts.

Reply:
Removed this code (`getOrCreateContainerProcessorSegment()`) from `StreamSegmentContainer` and moved it to `ContainerEventProcessorImpl`. Timeout is passed via configuration in its new location.